app/src/main/java/com/fstyle/structure_android/screen/main/MainPresenter.java

```diff
@@ -54,17 +51,8 @@ public void searchUsers(int limit, String keyWord) {
                 .searchUsers(limit, keyWord)
                 .subscribeOn(Schedulers.io())
                 .observeOn(AndroidSchedulers.mainThread())
-                .subscribe(new Action1<List<User>>() {
-                    @Override
-                    public void call(List<User> users) {
-                        mMainViewModel.searchUsersSuccess(users);
-                    }
-                }, new Action1<Throwable>() {
-                    @Override
-                    public void call(Throwable throwable) {
-                        mMainViewModel.searchError(throwable);
-                    }
-                });
+                .subscribe(users -> mMainViewModel.searchUsersSuccess(users),
+                        throwable -> mMainViewModel.searchError(throwable));
```

Comment:
ERROR: 'lambda arguments' have incorrect indentation level 24, expected level should be 20.